docker/bin/push_all

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+#
+# Copyright 2015-2020 The OpenZipkin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+# This script pushes images to GitHub Container Registry (ghcr.io). When a release, they are also
+# pushed to Docker Hub (docker.io).
+set -ue
```

Comment:
I've tested this script locally, but to test travis, I need to merge first.

First, I'd wait and see if master pushed as it should.
Then, I'd re-publish 2.22.0 via docker-2.22.0